mungegithub/mungers/submit-queue.go

```diff
@@ -1331,6 +1335,10 @@ func (sq *SubmitQueue) handleGithubE2EAndMerge() {
 }
 
 func (sq *SubmitQueue) mergePullRequest(obj *github.MungeObject, msg, extra string) bool {
+	isMaster, _ := obj.IsForBranch(branchMaster)
```

Comment:
nit: just use the string literal `"master"` instead of the constant `branchMaster`. Its easier to read and the constant won't ever need to be updated like a label constant might need to be.